tests/Console/Command/AddPrefixCommandTest.php

```diff
@@ -138,23 +139,50 @@ public function test_get_version_menu(): void
         $this->fileSystemProphecy->isAbsolutePath(Argument::cetera())->shouldNotHaveBeenCalled();
     }
 
-    public function test_scope_the_given_paths(): void
+    /**
+     * @testWith [true, false, "output_dir_from_config"]
```

Comment:
Interesting, I was not aware of the `@testWith`. I would however prefer to move it in a data provider where it has the merit of being clear PHP code instead of comments